### PR TITLE
Ci cd workflows

### PR DESCRIPTION
**Pull request by imrajsinghsandhu**
Add build script that is invoked whenever there is a pull request to master branch, to ensure all microservices build without errors

### CODE REVIEW

**Comment by wengkit1** [comment]
lgtm!

**Review by wengkit1 — APPROVED** [review]

File: frontend/service.yaml
Comment (wengkit1) [inline frontend/service.yaml:1]: Is this file needed?

Comment (imrajsinghsandhu) [reply frontend/service.yaml:1]: Yeap its part of deployment to GKE

Comment (imrajsinghsandhu) [reply frontend/service.yaml:1]: Each deployed microservice will have a deployment.yaml and service.yaml
